### DISCUSSION TITLE
[CLOSED] Is this group called TOC?

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/oc#2] [General]
We copied TOC from the [Hyperledger TOC](https://toc.hyperledger.org/). 
Technical Oversight Committee.

If there are alternative names let's discuss. 

Until we officially form we don't have any formal mechanism of voting here so let's go with consensus, if the group agrees unanimously without objection to another name we can pick that.

Few points.

- Keeping it short, 3 letters max.
- Tactical instead of just Technical might give a broader remit to this group.
- I had considered Advisory "TAC" but I was concerned advice didn't show how much power this team should have, but then again does oversight really express that as well.
- It was mentioned that TOC also stands for our Theory Of Change so we already have a clash.
- We use the word Operations for all the functions the GSF Staff perform (back end, keeping the foundation running) so let's say that's  a reserved word and you can't use it!
- We could just keep on using TOC, it would make life easier and just allow us to move forward without concerns we'll have to revist and change references of TOC to XXX.

**UPDATE**

There we're no objections to renaming this group to the "Oversight Committee" from the "Technical Oversight Committee"

**Comment by jawache** [comment]
Some feedback I've received from a board member recently is to call this the OC rather than the TOC.
1- Clears up overlap with the Theory of Change. (TOC)
2- Removing the word Technical frees up the scope of this group to be broader (which was the intent, not just technical oversight, budget oversight, rule/process/procedure oversight)

Will create a PR to propose this change, i feel that might be a better medium for discussion.

**Reply by jawache** [reply]
Added this PR for the proposed naem change: https://github.com/Green-Software-Foundation/toc/pull/4

**Reply by vaughanknight** [reply]
Table of Contents etc, TOC is a fairly overloaded TLA it seems.  But also agree on dropping the "Technical" which I feel will cause confusion.

**Reply by Willmish** [reply]
+1 on OC (Oversight Committee) - TOC too many overlaps (when @Sealjay  mentioned it too me, I actually asked if he meant Theory Of Change, as I missed the GSF Leadership call two weeks ago)
